src/main/resources/form_templates.json

```diff
@@ -1540,98 +1540,112 @@
   "formCategory" : "INSEW2016_VL",
   "fee" : "",
   "isAuthenticationRequired" : true,
-  "isFesEnabled" : true
+  "isFesEnabled" : true,
+  "messageTextIdList": [1,2]
```

Comment:
is this right? shouldn't it be [1,4] ?

Reply:
The difference between 2 and 4 is the 3rd paragraph but the word doc attached to the story doesn't have a 3rd paragraph. Should it be guidance4 or should it be a new one without a 3rd paragraph?

Reply:
you may need to pull on master in web.  That 3rd para has been moved as per https://github.com/companieshouse/efs-submission-web/pull/43/files
Though it had a condition on it anyway so the 3rd para wouldn't show for the LIQ forms.

Reply:
I see. I've made those changes